engines/sword25/util/lua/lmathlib.cpp

```diff
@@ -131,12 +131,12 @@ static int math_exp (lua_State *L) {
 }
 
 static int math_deg (lua_State *L) {
-  lua_pushnumber(L, luaL_checknumber(L, 1)/RADIANS_PER_DEGREE);
+  lua_pushnumber(L, Common::rad2deg<double>(luaL_checknumber(L, 1)));
```

Comment:
Even though it doesn't change anything, maybe use `Common::rad2deg<lua_Number>(...)` here and below since the code in lua seems to use that everywhere (probably so that it can easily be changed to use single precision floating points instead of double). Although because of the define, the old code would have been using double precision arithmetic (before casting back to float) even if lua_Number had been defined to float.